lms/services/canvas_api/client.py

```diff
@@ -299,13 +299,31 @@ class _ListGroupCategories(RequestsResponseSchema):
         id = fields.Integer(required=True)
         name = fields.Str(required=True)
 
-    def group_category_groups(self, group_category_id):
-        """List groups that belong to the group category/group set `group_category_id`."""
-        return self._client.send(
+    def group_category_groups(self, course_id, group_category_id):
+        """
+        List groups that belong to the group category/group set `group_category_id`.
```

Comment:
They are called group_categories in the API :facepalm: 

So if you are familiar with the canvas UI the use category here sound strange because `group set` is used all over the UI.

But in the other hand if you are looking at this module in isolation and reading canvas API docs `group_category` is use there (also on the endpoints URLs).